src/Simplex/Chat/Controller.hs

```diff
@@ -426,7 +427,7 @@ data ChatCommand
   | SetGroupTimedMessages GroupName (Maybe Int)
   | SetLocalDeviceName Text
   | ListRemoteHosts
-  | StartRemoteHost (Maybe (RemoteHostId, Bool)) -- Start new or known remote host with optional multicast for known host
+  | StartRemoteHost (Maybe (RemoteHostId, Bool)) (Maybe Text) (Maybe TransportHost) (Maybe Word16) -- Start new or known remote host with optional multicast for known host
```

Comment:
```
| StartRemoteHost {remoteHost_ :: Maybe (RemoteHostId, Bool), address :: Maybe RemoteCtrlAddress, port :: Maybe Word16}

data RemoteCtrlAddress = RemoteCtrlAddress {interface: String, address: String}
```

Then UI can pass the whole structure as is.

network-info returns String as interface, why does it need to be Text here? I don't think explicit usage of Text is justified, for short strings it can be counterproductive